components/DispatchAction.coffee

```diff
@@ -26,29 +26,25 @@ exports.getComponent = ->
   c.outPorts.add 'handle',
     datatype: 'all'
     addressable: true
+  c.outPorts.add 'handling',
+    datatype: 'integer'
```

Comment:
What is this for? Could use a description

Reply:
This is where we send the chosen handle index. Mostly just for feature-compatibility vs. routers/GroupRouter